helios-services/src/main/java/com/spotify/helios/rollingupdate/DeploymentGroupEventFactory.java

```diff
@@ -37,51 +36,68 @@
     HOSTS_CHANGED
   }
 
-  public Map<String, Object> rollingUpdateTaskSucceeded(final DeploymentGroup deploymentGroup,
-                                                        final RolloutTask task) {
+  private Map<String, Object> createEvent(final String eventType,
```

Comment:
Yea a separate commit sounds good. Something like the below?

DeploymentGroupEvent class:
* eventType
* timestamp
* deploymentGroup
* success
* reason
* failedTask

DeploymentGroupTaskEvent class inherits from DeploymentGroupTask and also includes:
* action
* target
* error
* errorCode
* metadata